rust-symcrypt/README.md

```diff
@@ -4,27 +4,36 @@ This crate provides friendly and idiomatic Rust wrappers over [SymCrypt](https:/
 
 This crate has a dependency on `symcrypt-sys`, which utilizes `bindgen` to create Rust/C FFI bindings.
 
-**Note:** As of version `0.1.3`, only `Windows AMD64`, and [`Linux Mariner`](https://github.com/microsoft/CBL-Mariner) are fully supported, with partial support for other linux distros such as `Ubuntu`.
+**Note:** As of version `0.2.0`, only `Windows AMD64`, and [`Linux Azure Linux`](https://github.com/microsoft/azurelinux) are fully supported, with partial support for other Linux distros such as `Ubuntu`.
 
-For ease of use, we have included a `symcrypttestmodule.dll` and `symcrypttestmodule.lib` in the `bin/amd64/` folder for users on Windows. This will only work on computers using an `AMD64 (x86_64)` architecture.
+## Quick Start Guide
 
-We have also included the required `libsymcrypt.so` files needed for Linux users. These `.so` files have been built for the [Mariner](https://github.com/microsoft/CBL-Mariner) distro, but have been tested and confirmed working on `Ubuntu 22.04.3` via WSL. Support for other distros aside from Mariner is not guaranteed. 
+### Windows:
+Download the `symcrypt.dll` and `symcrypt.lib` for you corresponding CPU architecture from the [SymCrypt Releases Page](https://github.com/microsoft/SymCrypt/releases/tag/v103.4.2) and place them somewhere accessible on your machine.
 
-If you are using a different architecture, you will have to continue with the install and build steps outlined in the `BUILD.md` file.
+Set the required `SYMCRYPT_LIB_PATH` environment variable. You can do this by using the following command:
 
-## Quick Start Guide
+`setx SYMCRYPT_LIB_PATH "<your-path-to-symcrypt-lib-folder>"`
 
-### Windows:
-Copy the `symcrypttestmodule.dll` and `symcrypttestmodule.lib` from the `/bin/amd64` folder and place it into your `C:/Windows/System32` folder. 
+During runtime, Windows will handle finding all needed `dll`'s in order to run the intended program, this includes our `symcrypt.dll` file.
 
-For more information please see the `BUILD.md` file
+Here are 2 recommended options to ensure your `symcrypt.dll` is found by Windows during runtime, for more info please see [Dynamic-link library search order](https://learn.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-search-order).
+
+1. Put the `symcrypt.dll` in the same folder as your output `.exe` file. If you are doing development (not release), the common path will be: `C:\your-project\target\debug\`.
+2. Permanently add the `symcrypt.dll` path into your System PATH environment variable. Doing this will ensure that any project that uses the SymCrypt crate will be able to access `symcrypt.lib`
+
+**Note this is a temporary solution while we work on shipping SymCrypt out of the box on Windows.**

```

Comment:
This isn't really private but I don't know if we should mention it here just because we don't really have a timeline/release vehicle for it yet.